flexmeasures/data/models/generic_assets.py

```diff
@@ -137,6 +137,15 @@ def asset_type(self) -> GenericAssetType:
         ),
     )
 
+    @property
+    def offspring(self) -> list[GenericAsset]:
+        offspring = []
```

Comment:
For the docstring: returns a flattened list of all offspring, which is looked up recursively.

Reply:
Thanks for the suggestion :D